apps/src/templates/teacherDashboard/AddInitialStudentsView.jsx

```diff
@@ -0,0 +1,146 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from '@cdo/locale';
+import color from '../../util/color';
+import experiments from '../../util/experiments';
+import CardContainer from './CardContainer';
+import LoginTypeCard from './LoginTypeCard';
+
+const style = {
+  h1: {
+    fontFamily: '"Gotham 7r", sans-serif',
+    fontWeight: 'normal',
+    fontSize: 32,
+    lineHeight: 1.5,
+    color: color.dark_charcoal,
+  },
+  h2: {
+    fontFamily: '"Gotham 4r", sans-serif',
+    fontWeight: 'normal',
+    fontSize: 24,
+    lineHeight: 2,
+    color: color.dark_charcoal,
+    margin: '10px 0',
+  },
+  h3: {
+    fontFamily: '"Gotham 5r", sans-serif',
+    fontWeight: 'normal',
+    fontSize: 16,
+    lineHeight: 2,
+    color: color.dark_charcoal,
+  },
+};
+
+/**
+ * View shown to a teacher when beginning to add students to an empty section.
+ * Lets the teacher decide whether to use word/picture logins, have students
+ * manage their own accounts via email/oauth, or to sync students with an
+ * external service like Microsoft Classroom or Clever.
+ */
+class AddInitialStudentsView extends Component {
+  static propTypes = {
+    sectionName: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const {sectionName} = this.props;
+    const googleClassroom = experiments.isEnabled('googleClassroom');
+    const microsoftClassroom = experiments.isEnabled('microsoftClassroom');
+    const clever = experiments.isEnabled('clever');
+    const anyThirdParty = googleClassroom || microsoftClassroom || clever;
+
+    return (
+      <div>
+        <h1 style={style.h1}>
```

Comment:
Yeah, I saw that, and I definitely had to override some application.css styles to make this work.

There's a downside to moving away from semantic tags though - it can impact accessibility.  Although that's never been our strong suit...

I wonder what the best of both worlds looks like.